tooling/api/src/event.ts

```diff
@@ -12,7 +12,7 @@
 import * as eventApi from './helpers/event'
 import type { EventCallback, UnlistenFn, Event } from './helpers/event'
 
-export type EventName = TauriEvent | string
+export type EventName = `${TauriEvent}`
```

Comment:
We also need to allow `string` type to be passed, `TauriEvent` is just for auto-completion purposes and not enforce a variant of it. I think we should be using this little trick from type-fest's [`LiteralUnion`](https://github.com/sindresorhus/type-fest/blob/main/source/literal-union.d.ts) instead

```suggestion
export type EventName = `${TauriEvent}` | (string & Record<never, never>)
```